cypress/e2e/specialFeatures/liteSiteWeight/assertions/liteSiteWeight.js

```diff
@@ -0,0 +1,62 @@
+import interceptGetRequests from '../helpers/interceptGetRequests';
+import getTotalPageSize from '../helpers/getTotalPageSize';
+
+const MAX_PAGE_WEIGHT_KB = 100;
+
+const roundTo2Decimals = num => {
+  return parseFloat(num.toFixed(2));
+};
```

Comment:
This might need to be moved into the helpers if it's used here, and in the assertions